### DISCUSSION TITLE
Mega Menu in Elementor with dynamic content loading upon hovering or clicking on the mega menu tab.

### DISCUSSION THREAD

**Discussion by DanielCyran** [elementor/elementor#27263] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

It will speed up the operation of stores/websites with a large mega menu with lots of content

### Describe the solution you'd like.

Firstly, I would like to thank the Elementor team for the recent updates that have improved the speed of websites created using Elementor <3

I have an idea for further improvement in terms of faster website performance:

The content of the mega menu should load upon hovering or clicking on a specific mega menu tab, so that pages load faster and the size of the DOM in PageSpeed is significantly reduced.

### Describe alternatives you've considered.

I have no alternatives :D only elementor <3

### Additional context

The speed of websites/shops affects their positioning and sales <3

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by nicholaszein** [comment]
Hi @DanielCyran!

Thank you for contributing with your ideas! 🙏 

The Mega Menu already triggers the content on Hover or On Click of the menu item. Can you please elaborate on what exactly you want that is different on your suggestion?

Kind regards
